hyperspy/misc/eds/utils.py

```diff
@@ -243,6 +247,72 @@ def take_off_angle(tilt_stage,
                                   + math.sin(a) * math.sin(c)))
 
 
+def xray_lines_model(elements=['Al', 'Zn'],
+                     beam_energy=200,
+                     weight_percents=[50, 50],
+                     energy_resolution_MnKa=130,
+                     energy_axis={'name': 'E', 'scale': 0.01, 'units': 'keV',
+                                  'offset': -0.1, 'size': 1024}
+                     ):
+    """
+    Generate a model of X-ray lines using a Gaussian epr x-ray lines.
+
+    The area under a main peak (alpha) is equal to 1 and weighted by the
+    composition.
+
+    Parameters
+    ----------
+    elements : list of strings
+        A list of chemical element symbols.
+    beam_energy: float
+        The energy of the beam in keV.
+    weight_percents: list of float
+        The composition in weight percent.
+    energy_resolution_MnKa: float
+        The energy resolution of the detector in eV
+    energy_axis: dic
+        The dictionary for the energy axis. It must contains 'size' and the
+        units must be 'eV' of 'keV'.
+
+    Example
+    -------
+    >>> s = utils_eds.simulate_model(['Cu', 'Fe'], beam_energy=30)
+    >>> s.plot()
+    """
+    from hyperspy._signals.eds_tem import EDSTEMSpectrum
+    from hyperspy.model import components
+    s = EDSTEMSpectrum(np.zeros(energy_axis['size']), axes=[energy_axis])
+    s.set_microscope_parameters(
+        beam_energy=beam_energy,
+        energy_resolution_MnKa=energy_resolution_MnKa)
+    s.add_elements(elements)
+    counts_rate = 1.
+    live_time = 1.
+    if weight_percents is None:
+        weight_percents = [100] * len(elements)
+    m = s.create_model()
+    for i, (element, weight_percent) in enumerate(zip(
+            elements, weight_percents)):
```

Comment:
`zip` crops to the output to the shortest list. Maybe we want to check that the lengths correspond first, and raise a `ValueError` if they disagree?